client/state/data-layer/wpcom-http/utils.js

```diff
@@ -92,7 +123,11 @@ export const dispatchRequest = ( initiator, onSuccess, onError, options ) => ( s
 
 	const data = getData( action );
 	if ( data ) {
-		return onSuccess( store, action, data );
+		const [ isValid, response ] = fromApi( data );
```

Comment:
NIT: `response` may be a bit misleading as it sounds like something straight out of the API, which may or may not be the case if I'm reading this correctly.

Reply:
`data`? I was looking for a way to say, "this is the response data you asked for"